minutes/meeting-202001019-4.md

```diff
@@ -0,0 +1,128 @@
+# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC
+
+Attendees
+
+Present (IRC)
+liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert
+
+* Chair: Liang Hai
+* Scribe: Simon Cozens
+
+## Resolutions
+
+None
+
+## Actions
+
+* Ask W3C for a dedicated video meeting space.
+* Discuss meeting time scheduling on GitHub issue.
+* Research setting up IRC/github-bot.
+* @scottkellum, please post more notes from implementation responsive text prototypes.
+
+## Attendance
+
+Vladimir Levantovsky, Liang Hai, Simon Cozens, Andrew Glass, Anthrotype, Bobby de Vos, Caleb Maclennan, Chris Chapman, David Singer, David Corbett, Duncan MacMichael, John Hudson, Johnathan Kew, Laurence, Mehmet Oğuz Derin, Nat McCully, Nate Willis, Ned Holbrook, Norbert, Renzhi Li, Simon Cozens, Toshiya Suzuki, Zachary Scheuren
```

Comment:
```suggestion
Andrew Glass, Bobby de Vos, Caleb Maclennan\*, Chris Chapman\*, Cosimo Lupo\*, David Corbett\*, David Singer\*, Duncan MacMichael, Greg Hitchcock, John Hudson\*, Johnathan Kew\*, Laurence Penney\*, Liang Hai\*, Mehmet Oğuz Derin\*, Nat McCully\*, Nate Willis\*, Ned Holbrook\*, Norbert Lindenberg, Peter Constable\*, Renzhi Li, Scott Kellum\*, Simon Cozens\*, Toshiya Suzuki\*, Vladimir Levantovsky\*, Zachary Scheuren\*

\* Formal participants of the FTCG.

The IRC channel recorded the following nicks: liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert
```

Changes:

- Added the following names according to my Zoom screenshots: Scott Kellum, Greg Hitchcock, Peter Constable
- Normalized the following names: Anthrotype → Cosimo Lupo, Laurence → Laurence Penney, Norbert → Lindenberg
- Removed a duplication of Simon’s name.
- Marked formal attendees.
- Applied a dumb alphabetical order.
- Incorporated the IRC attendance info.